### ISSUE TITLE
Doesn't work in nw.js since 0.19.26

### ISSUE THREAD

**Issue by Klaster1** [systemjs/systemjs#1290]
The cbfb7f724d74cc333783eda19e03c5ce3e1505c4 breaks nw.js. support. Any import throws an error:

`System.import('text')`

```
system.src.js:123 Uncaught (in promise) Error: chrome-extension://gaapadgjfilciikafglinmihgeihpcod/jspm_packages/github/systemjs/plugin-text@0.0.7.js:3
	})(System, System);
	   ^
	ReferenceError: System is not defined
	    at chrome-extension://gaapadgjfilciikafglinmihgeihpcod/jspm_packages/github/systemjs/plugin-text@0.0.7.js:3:4
	    at Object.exports.runInThisContext (vm.js:54:17)
	Evaluating chrome-extension://gaapadgjfilciikafglinmihgeihpcod/jspm_packages/github/systemjs/plugin-text@0.0.7.js
	Error loading chrome-extension://gaapadgjfilciikafglinmihgeihpcod/jspm_packages/github/systemjs/plugin-text@0.0.7.js
```

The same issue was reported here https://github.com/nwjs/nw.js/issues/4838 too. I've tried to revert mentioned cbfb7f724d74cc333783eda19e03c5ce3e1505c4 to see if the trouble's gone for 0.19.29 and it ineed works as intended afterwards.

**Comment by guybedford** [comment]
Thanks for letting me know about this. I've commented on that issue and included the above fix in master for the next release.

**Comment by guybedford** [comment]
Released in 0.19.30.
